File: rxpanda/src/main/java/com/pandaq/rxpanda/entity/IApiData.java
package com.pandaq.rxpanda.entity;

/**
 * Created by huxinyu on 2019/6/18.
 * Email : panda.h@foxmail.com
 * <p>
 * Description :
 */
public interface IApiData<T> {

    Long getCode();

    String getMsg();

    T getData();

    boolean isSuccess();
}

File: rxpanda/src/main/java/com/pandaq/rxpanda/entity/IApiData.kt
package com.pandaq.rxpanda.entity

/**
 * Created by huxinyu on 2019/6/18.
 * Email : panda.h@foxmail.com
 *
 *
 * Description :
 */
interface IApiData<T> {
    val code: String?
    val msg: String?
    val data: T?

    fun isSuccess():Boolean
}